nwbwidgets/test/test_plot_func.py

```diff
@@ -0,0 +1,155 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+from nwbwidgets.base import fig2widget, vis2widget, show_subject, import show_dynamic_table, dict2accordion
+from ipywidgets import widgets
+from pynwb import TimeSeries
+import pandas as pd
+from hdmf.common import DynamicTable
+from pynwb.behavior import Position, SpatialSeries, BehavioralEvents
+from nwbwidgets.view import default_neurodata_vis_spec
+from nwbwidgets.behavior import show_position, show_behavioral_events, show_spatial_series_over_time, show_spatial_series
+from pynwb.misc import AnnotationSeries
+from nwbwidgets.misc import show_annotations
+from utils.timeseries import get_timeseries_tt
+
+
+
+def test_fig2widget():
+    
+    data = np.random.rand(160,3)
+    
+    fig=plt.figure(figsize=(10, 5))
+    plt.plot(data)
+    
+    isinstance(fig2widget(fig), widgets.Widget)
+
+
+def test_vis2widget_input_widget():
+        
+    wg = widgets.IntSlider(
+    value=7,
+    min=0,
+    max=10,
+    step=1,
+    description='Test:',
+    disabled=False,
+    continuous_update=False,
+    orientation='horizontal',
+    readout=True,
+    readout_format='d')
+
+    vis2widget(wg)
+
+
+def test_vis2widget_input_figure():
+    
+    data = np.random.rand(160,3)
+    
+    fig=plt.figure(figsize=(10, 5))
+    plt.plot(data)
+    
+    vis2widget(fig)
+
+
+def test_show_subject():
+    
+    data = np.random.rand(160,3)
+    ts = TimeSeries(name='test_timeseries', data=data, unit='m', starting_time=0.0, rate=1.0)
+
+    
+    show_subject(ts)

```

Comment:
you are running `show_subject` on a `TimeSeries` object. `show_subject` is meant to be run on a `Subject` object.